From fe9e48cf1650c7674c7ac8350455e1f5b111838a Mon Sep 17 00:00:00 2001
From: WarmUpTill <19472752+WarmUpTill@users.noreply.github.com>
Date: Wed, 28 May 2025 21:42:08 +0200
Subject: [PATCH] Add option to not link date-tz even if the source code exists

---
 plugins/twitch/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/plugins/twitch/CMakeLists.txt b/plugins/twitch/CMakeLists.txt
index 0667809f2..d1d67b68d 100644
--- a/plugins/twitch/CMakeLists.txt
+++ b/plugins/twitch/CMakeLists.txt
@@ -31,7 +31,8 @@ if(NOT ZLIB_FOUND)
 endif()
 
 set(DATE_LIB_DIR "${ADVSS_SOURCE_DIR}/deps/date")
-if(EXISTS "${DATE_LIB_DIR}/CMakeLists.txt")
+if(EXISTS "${DATE_LIB_DIR}/CMakeLists.txt"
+   AND NOT DISABLE_TWITCH_TIMESTAMP_VERIFICATION)
   set(BUILD_TZ_LIB ON)
   if(OS_WINDOWS)
     if(CURL_FOUND AND TARGET CURL::libcurl)
